website/docs/docs/build/unit-tests.md

```diff
@@ -28,6 +28,7 @@ Starting in dbt Core v1.8, we have introduced an additional type of test to dbt
 - If your model has multiple versions, by default the unit test will run on *all* versions of your model. Read [unit testing versioned models](/reference/resource-properties/unit-testing-versions) for more information.
 - Unit tests must be defined in a YML file in your `models/` directory.
 - Table names must be [aliased](/docs/build/custom-aliases) in order to unit test `join` logic.
+- All [`ref`](/reference/dbt-jinja-functions/ref) or [`source`](/reference/dbt-jinja-functions/source) references within the model must be included in the unit test configuration as`input`s otherwise you may encounter "node not found" errors during compilation.
```

Comment:
```suggestion
- Include all [`ref`](/reference/dbt-jinja-functions/ref) or [`source`](/reference/dbt-jinja-functions/source) model references in the unit test configuration as `input`s to avoid "node not found" errors during compilation.
```